scripts/asm-installer/install_asm

```diff
@@ -166,16 +166,18 @@ OPTIONS:
   -p|--project_id        <ID>         The GCP project ID
   -m|--mode              <MODE>       The type of installation to perform.
                                       Allowed values for <MODE> are
-                                      {install|migrate}. Choose --mode=install
-                                      for a new ASM installation or
-                                      --mode=migrate for an existing OSS
-                                      installation to be migrated to ASM.
-                                      MODE=install defaults to using Mesh CA
-                                      as the certificate authority.
-  -c|--ca                <CA>         The type of certificate authority to be
-                                      used. Allowed values for <CA> are
-                                      {citadel|mesh_ca}.
-
+                                      {install|migrate}. Choose MODE=install
+                                      for a new ASM installation or MODE=migrate
+                                      for an existing OSS installation to be
+                                      migrated to ASM. MODE=install defaults to
```

Comment:
Can we change "MODE=install" to "--mode install"? Caps vs not is optional, but in the CA field we use lowercase so we should be standard. We shouldn't use an equal sign since the script doesn't support using it.

Reply:
Sure. I originally tried this but it led to an unfortunate line break which made things a bit confusing. But I'll do it that way, since it is in fact correct.